src/rprblender/utils/sequence.py

```diff
@@ -0,0 +1,54 @@
+#**********************************************************************
+# Copyright 2020 Advanced Micro Devices, Inc
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#********************************************************************
+import os
+import pathlib
+
+from . import logging
+log = logging.Log(tag='sequence')
+
+
+# accept up to this number of leading zeroes in frame number in filename
+MAX_FRAME_NUMBER_LEADING_ZEROS = 6
+
+
+def get_sequence_frame_file_path(source_path, frame_number):
```

Comment:
I think it can be moved to utils, no need new module for just one help function.